tools/system_libs.py

```diff
@@ -1385,6 +1386,7 @@ class libasan_rt_wasm(SanitizerLibrary):
 # in JS as it's more efficient that way).
 class libstandalonewasm(MuslInternalLibrary):
   name = 'libstandalonewasm'
+  depends = ['libc++abi']
```

Comment:
just to be sure, this won't increase code size by default, correct? only used files from that library will actually be included?

Reply:
Yes, as a rule, add more libraries is free as long you don't reference stuff in them.  

This is less true for fastcomp where it costs a little in link time, but DCE should mean it doesn't cost anything in the final binary.    But I think I can remove this line now.. let me try.